website/docs/docs/cloud/visual-editor.md

```diff
@@ -2,17 +2,17 @@
 title: "About the Visual Editor" 
 id: visual-editor       
 sidebar_label: "About the Visual Editor" 
-description: "The Visual Editor enables analysts to quickly create and visualize dbt models through a visual, drag-and-drop experience inside of dbt Cloud." 
+description: "The Visual Editor enables you to quickly create and visualize dbt models through a visual, drag-and-drop experience inside of dbt Cloud." 
 pagination_next: "docs/cloud/visual-editor-interface"
 pagination_prev: null
 ---
 
 import Prerequisites from '/snippets/_visual-editor-prerequisites.md';
 
-# About the Visual Editor <Lifecycle status='beta'/> 
+# About the Visual Editor <Lifecycle status='beta, enterprise'/> 
```

Comment:
Do you want the Enterprise pill on the other two pages as well?

Reply:
Yeah, good call. We should put it on all three